src/celeritas/ext/ScopedRootErrorHandler.cc

```diff
@@ -64,9 +66,12 @@ void RootErrorHandler(Int_t rootlevel,
         auto log_msg = ::celeritas::world_logger()({"ROOT", 0}, level);
         if (location)
         {
-            log_msg << color_code('x') << location << color_code(' ') << ": ";
+            log_msg << color_code('x') << location << color_code(' ');
+        }
+        if (msg && msg[0] != '\0')
+        {
+            log_msg << ": " << msg;
```

Comment:
The `LoadPCM` failure message is empty, making the trailing colon confusing.

Reply:
The change it self is fine and make sense.  That there an error message with no message is odd/surprising :(. (At least for `LoadPCM` I don't see how it happened)